runbooks/runbook.md

```diff
@@ -0,0 +1,110 @@
+# UPP - Annotations Mapper
+
+Annotations Mapper processes metadata about content that comes from the QMI system (v1 metadata).
+## Code
```

Comment:
Doesn't seem to break the parsing, but a new line here would make the file more readable.